src/search/search.cpp

```diff
@@ -126,14 +126,14 @@ void PrintPacket(char* data, int size)
         if (((y + 1) % 16) == 0)
         {
             message[48] = '\n';
-            printf(message);
+            puts(message);
```

Comment:
Note, puts appends \n at the end  of string. Either use printf("%s", message); or fputs()